Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -89,6 +84,21 @@ final class SettingsViewController: UIViewController {
       .observeValues { [weak self] link in self?.goToAppStore(link: link) }
   }
 
+  private func leftBarButtonItem() -> UIBarButtonItem {
```

Comment:
This can be:
```suggestion
  private func leftBarButtonItem() -> UIBarButtonItem {
    return UIBarButtonItem(
      image: UIImage(named: "icon--cross"),
      style: .plain,
      target: self,
      action: #selector(closeButtonPressed)
    )
    |> \.accessibilityLabel %~ { _ in Strings.Dismiss() }
    |> \.width .~ 44
  }
```

Reply:
Wow, Github really messed up here..had to manually fix it later on :)